### ISSUE TITLE
Github Stats card: Total commits count represents last 365 days, not current year as indicated

### ISSUE THREAD

**Issue by harry-graham** [anuraghazra/github-readme-stats#2501]
Discussed in https://github.com/anuraghazra/github-readme-stats/discussions/2498.

## Overview

On the Github Stats card, the `Total Commits (2023)` value suggests that it represents the total number of commits made by the user in 2023.

However, the count actually represents the total commits by the user in the last 365 days.

### Github Stats cards for reference

With `count_private: false`:

[![My GitHub stats](https://github-readme-stats.vercel.app/api?username=harry-graham&hide=stars&show_icons=true)](https://github.com/anuraghazra/github-readme-stats)

```
[![My GitHub stats](https://github-readme-stats.vercel.app/api?username=harry-graham&hide=stars&show_icons=true)](https://github.com/anuraghazra/github-readme-stats)
```

With `count_private: true`:

[![My GitHub stats](https://github-readme-stats.vercel.app/api?username=harry-graham&hide=stars&count_private=true&show_icons=true)](https://github.com/anuraghazra/github-readme-stats)

```
[![My GitHub stats](https://github-readme-stats.vercel.app/api?username=harry-graham&hide=stars&count_private=true&show_icons=true)](https://github.com/anuraghazra/github-readme-stats)
```

### Querying the API

With date range of this year only:

![image](https://user-images.githubusercontent.com/35763376/216847656-4089fe43-1313-427b-98da-d11666cb13b2.png)

With date range of last 365 days, which matches the Github Stats cards shown above (with small difference likely due to caching):

![image](https://user-images.githubusercontent.com/35763376/216847663-4fff9d80-f2d3-4b23-b05f-b5535d04c049.png)

**Comment by rickstaa** [comment]
@harry-graham, thanks for your bug report. This is a known issue and has been discussed at https://github.com/anuraghazra/github-readme-stats/issues/1053. It is solved when https://github.com/anuraghazra/github-readme-stats/pull/2107 is merged. Please upvote (i.e. 👍) https://github.com/anuraghazra/github-readme-stats/issues/1053 and/or https://github.com/anuraghazra/github-readme-stats/pull/2107 to speed up the merging process.

**Comment by rickstaa** [comment]
> @harry-graham, thanks for your bug report. This is a known issue and has been discussed at #1053. It is solved when #2107 is merged. Please upvote (i.e. 👍) #1053 and/or #2107 to speed up the merging process (see #1935).

You can ofcourse already use https://github.com/anuraghazra/github-readme-stats/pull/2107 on your own fork until it gets merged in the main branch.
